src/openclsim/model/move_activity.py

```diff
@@ -81,6 +81,8 @@ def main_process_function(self, activity_log, env):
             speed the mover should sail. for example, engine_order=0.5
             corresponds to sailing at 50% of max speed
         """
+
+        print("mover", self.mover.name, "destination", self.destination.name)
```

Comment:
remaining dev statement? If you want to keep it is should be done with the logger, but I preffer not

Reply:
Removed